master/usage/policy-example.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Realistic Policy Demo
```

Comment:
I think this title is a bit off - we've got a bunch of demos in our documentation that are "realistic", but this implies that this one is somehow more realistic?